src/SelectVideo/SelectVideo.tsx

```diff
@@ -38,17 +45,24 @@ const SelectVideo: React.FunctionComponent<RouteComponentProps<IParams>> = () =>
         alignItems={`center`}
       >
         <CarouselContainer>
-          Youtube-Box  
+          <MediaServiceProviderBox>
+            <img width="122" src={YoutubeLogo} alt={'youtube'}/>
```

Comment:
Magic number

Reply:
Any suggestion on how should I define so many different widths and heights as constants?

Reply:
Once you collect them in the constants file it would be easier.
You'd have to work out the naming on case by case basis.

Some constants can reside in route-level components if needed.

Reply:
Moved numbers to constants inside route-level components.
I hope this works.